addon/comment/continuecomment.js

```diff
@@ -9,6 +9,8 @@
   else // Plain browser env
     mod(CodeMirror);
 })(function(CodeMirror) {
+  var nonspace = /\S/g;
+  var repeat = String.prototype.repeat || function (n) { return Array(n).join(this); };
```

Comment:
The fallback doesn't quite work—should be `Array(n + 1)` since join will only join _between_ the elements.

Reply:
Indeed!
Fixed.